src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -35,6 +35,9 @@ exception. The attribute value will be set to full name of exception type.
 * Fixed `network.protocol.version` attribute values to match the specification.
   ([#5007](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5007))
 
+* Removed filtering of metrics when the request path contained `/metrics`.
```

Comment:
It won't hurt to mention that this is very common in Prometheus scraping, so hint that more measurements will be included in the final metric with this change. The "more" is likely negligible as prom's default scrap is 15secs only.

Please reword, above is just some suggestions!

Reply:
Updated. is this better?